models/yang/sonic/sonic-wred-profile.yang

```diff
@@ -0,0 +1,79 @@
+module sonic-wred-profile {
+    namespace "http://github.com/Azure/sonic-wred-profile";
+    prefix wrd;
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC WRED_PROFILE";
+
+    revision 2019-05-15 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-wred-profile {
+        container WRED_PROFILE {
+            list WRED_PROFILE_LIST {
+                key "name";
+
+                leaf name{
+                    type string;
+                }
+
+                leaf yellow_min_threshold {
+                    type uint64;
```

Comment:
Define constraint like range of values or special value etc. Take care in all places applicable.